x-pack/test/ui_capabilities/security_and_spaces/tests/nav_links.ts

```diff
@@ -57,7 +57,8 @@ export default function navLinksTests({ getService }: FtrProviderContext) {
                 'enterpriseSearchContent',
                 'enterpriseSearchAnalytics',
                 'appSearch',
-                'workplaceSearch'
+                'workplaceSearch',
+                'guidedOnboarding'
```

Comment:
```suggestion
                'guidedOnboardingFeature'
```

Reply:
Other alternative is to change the feature ID to 'guidedOnboarding'. May be more consistent with other IDs.

Reply:
Alternative - change the ID to 'guidedOnboarding'. This may be more consistent with other feature ID's.